webdriver/tests/bidi/browser/create_user_context/proxy.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+
+from .. import get_user_context_ids
+
+pytestmark = pytest.mark.asyncio
+
+
+@pytest.fixture
+def create_user_context_with_proxy_and_assert_it_is_created(bidi_session,
+        create_user_context):
+    async def create_user_context_with_proxy_and_assert_it_is_created(proxy):
+        user_context = await create_user_context(proxy=proxy)
+
+        # TODO: check the parameter is respected.
+
+        assert user_context in await get_user_context_ids(bidi_session)
+
+    return create_user_context_with_proxy_and_assert_it_is_created
+
+
+async def test_system(create_user_context_with_proxy_and_assert_it_is_created):
+    await create_user_context_with_proxy_and_assert_it_is_created({
+        "proxyType": "system"
+    })
+
+
+async def test_autodetect(
+        create_user_context_with_proxy_and_assert_it_is_created):
+    await create_user_context_with_proxy_and_assert_it_is_created({
+        "proxyType": "autodetect"
+    })
+
+
+async def test_direct(create_user_context_with_proxy_and_assert_it_is_created):
+    await create_user_context_with_proxy_and_assert_it_is_created({
+        "proxyType": "direct"
+    })
+
+
+@pytest.mark.parametrize("ftpProxy", [None, "127.0.0.1:21"])
```

Comment:
We don't support the ftp proxy anymore, because Firefox ftp URLs are deprecated. Looks like they are deprecated for Chrome and Safari, so not sure if it makes sense to test.

Reply:
It's a part of protocol, so I believe we should remove it from the protocol first.